org.eclipse.m2e.pde.ui/src/org/eclipse/m2e/pde/ui/target/editor/internal/ClipboardUtils.java

```diff
@@ -0,0 +1,70 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Patrick Ziegler and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *      Patrick Ziegler - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.m2e.pde.ui.target.editor.internal;
+
+import java.text.MessageFormat;
+import java.util.Collections;
+import java.util.List;
+
+import org.eclipse.core.runtime.Platform;
+import org.eclipse.m2e.pde.target.MavenTargetDependency;
+import org.eclipse.m2e.pde.ui.target.editor.ClipboardParser;
+import org.eclipse.m2e.pde.ui.target.editor.MavenTargetLocationWizard;
+import org.eclipse.m2e.pde.ui.target.editor.Messages;
+import org.eclipse.swt.dnd.Clipboard;
+import org.eclipse.swt.dnd.TextTransfer;
+import org.eclipse.swt.widgets.Display;
+
+public abstract class ClipboardUtils {
```

Comment:
Can this static method be moved to `ClipboardParser` instead of having a util class?